client/src/components/mainContent.jsx

```diff
@@ -90,12 +104,26 @@ const MainContent = () => {
                         {meeting.description}
                       </Typography>
                     </div>
+                    <Divider className={classes.DividerInCardContent}/>
+                    <div className={classes.BelowDividerinCardContent}>
```

Comment:
similar, we should reserve PascalCase for components and use snakeCase for classes as well
```suggestion
                    <div className={classes.belowDivierInCardContent}>
```